test/src/e2e_vm_tests/test_programs/should_fail/u256/.gitignore

```diff
@@ -0,0 +1,2 @@
+out
+target
```

Comment:
Is this local `.gitignore` file necessary? I can't recall seeing those in other PRs